package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "",
+  "version": "",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
```

Comment:
Usually, `start` is used, because `npm start` is a common practice. `serve` is not a pre-defined script in `npm`, people have to use `npm run serve`.